src/bundle/Resources/views/themes/admin/content_type/create.html.twig

```diff
@@ -51,8 +51,14 @@
                             {% block left_column %}
                                 {% set form_field_options = { row_attr: { class: 'ibexa-form-field' } } %}
 
-                                {{ form_row(form.name, form_field_options) }}
-                                {{ form_row(form.identifier, form_field_options) }}
+                                {{ form_row(form.name, {
+                                    row_attr: { class: 'ibexa-form-field' },
```

Comment:
It would be better to reuse `form_field_options`, you can use merge filter to add additional attr.